src/tests/roc_sndio/target_sox/test_pump.cpp

```diff
@@ -24,7 +24,10 @@ namespace sndio {
 
 namespace {
 
-enum { MaxBufSize = 8192, FrameSize = 512, SampleRate = 44100, ChMask = 0x3 };
+enum { MaxBufSize = 8192, SampleRate = 44100, ChMask = 0x3 };
+
+const core::nanoseconds_t MaxBufDuration =
+    MaxBufSize * core::Second / (SampleRate * packet::num_channels(ChMask));
```

Comment:
I suggest to rename MaxBufSize/MaxBufDuration to BufSize/BufDuration and reduce size from 8192 to 512 to keep tests a bit more lightweight

Reply:
done.